package.json

```diff
@@ -30,11 +33,22 @@
   },
   "devDependencies": {
     "@babel/core": "^7.20.0",
+    "@commitlint/cli": "^19.3.0",
+    "@commitlint/config-conventional": "^19.2.2",
     "@types/react": "~18.2.45",
     "eslint": "^8.57.0",
     "eslint-config-expo": "^7.0.0",
+    "husky": "^9.0.11",
     "prettier": "^3.2.5",
     "typescript": "~5.3.3"
   },
+  "husky": {
```

Comment:
I go back and forth on husky but I get it's required for commitlint basically

Reply:
I think some of it for me is using the right hook for the right tool and not blocking dev progress.. like idc if you mess up your branch, but don't do things to mess up the main branch in the repo